android/app/build.gradle

```diff
@@ -44,6 +44,12 @@ android {
             returnDefaultValues = true
         }
     }
+    packagingOptions {
+        exclude 'META-INF/LICENSE.txt'
```

Comment:
Do we really need this fakes (please let me know if they are not fakes) to be merged?

Reply:
Yes, this is required for gradle 2.2.3 not to throw a merge error.